File: python/Excel Sheet Column Number.py
class Solution(object):
    def titleToNumber(self, s):
        base = 1
        answer = 0
        for i in range(len(s) - 1, -1, -1):
            answer += (ord(s[i]) - ord('A') + 1) * base
            base *= 26
        return answer
        

File: cpp/Excel Sheet Column Number.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int base = 1, num = 0;
        for (int i = 0; i < s.length(); i++) {
            num = num * 26 + (s[i]-'A'+1);
        }
        return num;
    }
};
